SourceX/display_fwd.h

```diff
@@ -0,0 +1,30 @@
+namespace disp {
+
+// Initializes the size of various graphical elements of the game. This is called within dvl::SpawnWindow.
```

Comment:
I'm not super fond of this. Instead, I have opted to remove the dynamic sizes from static values. This was also why I wanted to do the UI refactoring first, so I understand why you didn't go for that approch initially.

Reply:
An added benefit of this approach is that it should be relatively easy to allow for the resolution to be changed (either via options, or resizing the window) at runtime at a later point.